### PR TITLE
(PC-12864) feat(InputError): handle accessibility on input error

### PR DESCRIPTION
**Pull request by marionnousvalentin**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12864

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets.
- [x] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |         
image
         |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/libs/accessibility/ErrorMessage.web.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export const ErrorMessage: React.FC = (props) => {
+  return <div role="alert">{props.children}</div>
+}
```
Comment (Lucasbeneston) [inline src/libs/accessibility/ErrorMessage.web.tsx:5]: Je crois que tu peux plutôt utiliser une view avec l'accessibilityRole="alert" que ls div. On n'utilise un max de composant natif même dans un .web

File: src/libs/accessibility/ErrorMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React, { Fragment } from 'react'
+
+export const ErrorMessage: React.FC = (props) => {
+  return <Fragment>{props.children}</Fragment>
+}
```
Comment (Lucasbeneston) [inline src/libs/accessibility/ErrorMessage.tsx:5]: export { Fragment as ErrorMessage } from 'react'

File: src/libs/accessibility/ErrorMessage.web.tsx (hunk 1 above)
Comment (Lucasbeneston) [reply src/libs/accessibility/ErrorMessage.web.tsx:5]: En faisant attention a ce que le style de la view corresponde bien

Comment (Lucasbeneston) [reply src/libs/accessibility/ErrorMessage.web.tsx:5]: Mais a voir si c'est pas mieux d'utiliser une div directement comme l'équivalent natif utilise un fragment. @kopax

Comment (marionnousvalentin) [reply src/libs/accessibility/ErrorMessage.web.tsx]: Okep, je mets une view

**Review by Lucasbeneston — APPROVED** [review]
👍

**Pull request merged**